### ISSUE TITLE
Saves crash reports in the wrong directory

### ISSUE THREAD

**Issue by JJTech0130** [deltacv/EOCV-Sim#85]
**Describe the bug**
When EOCV-Sim tries to save a crash report, it does not append a slash to the current directory, leading to the report being saved to the parent directory.

For example, instead of saving the crash report as `crashreport-eocvsim-2022_09_16-10.41.04.log` in the `/Users/jamesgill/Downloads/` folder, it's saved as `Downloadscrashreport-eocvsim-2022_09_16-10.41.04.log` in `/Users/jamesgill`

**To Reproduce**
1. Run EOCV-Sim.jar in a situation where it will crash
2. Look for the crash report
3. (My specific crash is the inability to load OpenCV, but I'll open a separate issue for that)

**Expected behavior**
The crash report should be saved in the current directory, by appending the / in the proper place.
**Logs**
```
[10:55:50] [main/ERROR]: [EOCVSim] The sim will exit now as it's impossible to continue execution without OpenCV
[10:55:50] [main/INFO]: [CrashReport] Saved crash report to /Users/jamesgill/Downloadscrashreport-eocvsim-2022_09_16-10.55.50.log
```

**Desktop (please complete the following information):**
 - OS: macOS Sierra (10.12.6)
 - Processor architecture: x86_64
 - EOCV-Sim Version: 3.4.1
 - Attach logs if possible [e.g autogenerated crash report files in the same directory as the jar file or the console output]
[Downloadscrashreport-eocvsim-2022_09_16-10.55.50.log](https://github.com/deltacv/EOCV-Sim/files/9585196/Downloadscrashreport-eocvsim-2022_09_16-10.55.50.log)

**Comment by serivesmejia** [comment]
Thank you for reporting, I'll try to look into it and fix it asap. Make sure to open an issue for the actual crash report if possible though

**Comment by JJTech0130** [comment]
It seems like the actual crash is because of macOS being too old, I haven’t had time to investigate thoroughly yet

**Comment by serivesmejia** [comment]
Fixed in v3.4.2
